guide/cpu/cpu.md

```diff
@@ -0,0 +1,36 @@
+# The CPU
```

Comment:
Do you think this section could be combined with the instruction cycle?

Reply:
ooo yes! will make the changes.